### ISSUE TITLE
tests failing

### ISSUE THREAD

**Issue by joethorley** [poissonconsulting/fwapgr#42]
A couple of the fwapgr tests are failing.

The following query used to return the equivalent of data.frame with 2 columns and 1 row but now returns the equivalent of a data.frame with 1 column with no rows.

```
>   blk <- 356308001
>   drm2 <- 10000
>   properties <- c("area_ha")
>   df <- fwa_watershed_at_measure(blk, downstream_route_measure = drm2, properties = properties)
[1] "https://hillcrestgeo.ca/fwapg/functions/fwa_watershedatmeasure/items.json"
<request>
Options:
* useragent: http://github.com/poissonconsulting/fwapgr
$blue_line_key
[1] 356308001

$downstream_route_measure
[1] 10000

$limit
[1] 100

$offset
[1] 0

$bbox
NULL

$properties
[1] "area_ha"

$transform
NULL
```

```
[1] "{\"type\":\"FeatureCollection\",\"features\":[],\"numberReturned\":0,\"timeStamp\":\"2021-09-03T13:06:50-07:00\",\"links\":[{\"href\":\"https://www.hillcrestgeo.ca/fwapg/collections/fwa_watershedatmeasure/items\",\"rel\":\"self\",\"type\":\"application/json\",\"title\":\"This document as JSON\"},{\"href\":\"https://www.hillcrestgeo.ca/fwapg/collections/fwa_watershedatmeasure/items.html\",\"rel\":\"alternate\",\"type\":\"text/html\",\"title\":\"This document as HTML\"}]}"
```

**Comment by joethorley** [comment]
I've also reported in an email to @smnorris

**Comment by smnorris** [comment]
I've added the missing functions that FWA_WatershedAtMeasure depends on to the db - should be good now.

**Comment by joethorley** [comment]
Thanks Simon - that's fixed it but see #43
